README.md

```diff
@@ -229,6 +229,23 @@ To increase or decrease the logging level for the S3 plugin, add an `s3_resource
 },
 ```
 
+## Troubleshooting
+
+### Handling Reduce Your Request Rate Errors
+
+On operations that require a checksum calculation to an S3 object at rest, iRODS must read the object's data to calculate the checksum. The default buffer size for these reads is 1 MiB. Each of these read requests map to a separate GetObject request to S3 with a range of bytes. This can exceed the request rate that the S3 provider allows.
```

Comment:
```suggestion
On operations that require calculating a checksum calculation for an S3 object at rest, iRODS must read the object's data to calculate the checksum. The default buffer size for these reads is 1 MiB. Each of these read requests map to a separate GetObject request to S3 with a range of bytes. This can exceed the request rate that the S3 provider allows.
```

Reply:
Updated as listed above but left out the "calculation" after checksum.

Reply:
Oh, I didn't read the whole sentence. My bad...